### DISCUSSION TITLE
Example code with imports for 3.01?

### DISCUSSION THREAD

**Discussion by JamesTreleaven** [ezylang/EvalEx#337] [General]
It would be nice if there was a complete example with import statements and a pom.xml file somewhere for 3.01.  I have been using 2.5 for a long time but I am having a lot of trouble getting 3.01 going because I am guessing what my import statements should be.

When I do the following (in Scala):

import com.ezylang.evalex.Expression
val exp = new Expression("\"abc\" == \"abc\"")

I get the exception:
Caused by: java.lang.NoClassDefFoundError: com/ezylang/evalex/Expression

So a small complete example with import statements would really help me.

**Comment by RobertZenz** [comment]
>  Caused by: java.lang.NoClassDefFoundError: com/ezylang/evalex/Expression

Well that does sound like a project setup issue, namely you don't have EvalEx 3 on your classpath. The [README does container the necessary Maven statement for that](https://github.com/ezylang/EvalEx#maven), do you have that?

**Reply by JamesTreleaven** [reply]
I do have it and I believe that I have set it up correctly because it builds just fine. Normally if I have not configured maven properly it wont successfully build.

(My situation is a bit odd in that I am coding in Scala and using sbt to pull the appropriate jars from maven - but with every other maven library I am using if I dont have the maven-settings right I get an error at build time and not at execution time.)

**Reply by RobertZenz** [reply]
But it does not seem to be found at runtime, how are you launching your application?

**Comment by JamesTreleaven** [comment]
A colleague pointed out to me this morning that EvalEx 3 requires Java 11 but I was using Java 8.  So I think that explains everything.  Sorry for not checking that my dependencies were all fulfilled.
